Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -212,6 +220,7 @@ protected virtual void OnEnable()
 
         protected virtual void OnDisable()
         {
+            if (this.isActiveAndEnabled && Application.isPlaying) Debug.Log("Touch Disabled");
```

Comment:
Same as above: Looks like this is a left-over from debugging? Should be removed if it is.